lib/merchant_repository.rb

```diff
@@ -0,0 +1,44 @@
+class MerchantRepository
+  attr_reader :data,
+              :all
+
+  def initialize(data)
+    @data = data
+    @all = populate_repo
```

Comment:
Maybe we can match the syntax of creating an `all` method and replacing the variable
I.E `initialize`
```
@merchant_arr = populate_repo
```
`def all`
```
def all
 @merchant_arr
end
```

That way you can remove the `@` in each method you call it, referencing the method instead of the instance variable

Reply:
That makes sense Ben